packages/react/src/components/TeachingBubble/__snapshots__/TeachingBubble.test.tsx.snap

```diff
@@ -1531,6 +1539,7 @@ exports[`TeachingBubble renders TeachingBubbleContent with small headline correc
   tabIndex={-1}
 >
   <div
+    data-tabster="{\\"uncontrolled\\": {\\"completely\\": true}}"
```

Comment:
Why do some of the snapshots have `completely: true` but others don't?

I would have expected them all to have `completely: true` based on the hardcoded configuration.

Reply:
Ah, nvm, saw the other resolved conversation about this :)